coalib/settings/ConfigurationGathering.py

```diff
@@ -492,8 +499,16 @@ def gather_configuration(acquire_settings,
     aspectize_sections(sections)
     local_bears, global_bears = fill_settings(sections,
                                               acquire_settings)
+
+    only_global_bears = False
+    for sec, value in global_bears.items():
+        if value:
+            only_global_bears = True
+    for sec, value in local_bears.items():
+        if value:
+            only_global_bears = False
```

Comment:
1. You don't need `sec`, so instead of `items()`, you can iterate over `values()`.
2. You should break once `value == True`
3. You can use `any`:

```python
only_global_bears = any(global_bears.values()) and not any(local_bears.values())
```